docs/how-tos/deploy-ml.md

```diff
@@ -187,17 +344,30 @@ viam train logs --job-id=<JOB ID>
 {{% /tablestep %}}
 {{< /table >}}
 
+## Test your ML model
+
+{{<gif webm_src="/services/vision/mug-classifier.webm" mp4_src="/services/vision/mug-classifier.mp4" alt="A classification model run against an image containing a mug." max-width="250px" class="alignright">}}
+
+Once your model has finished training, you can test it with images in the cloud Viam cloud:
+
+1. Navigate to the [**DATA** tab](/services/data/view/) and click on the **Images** subtab.
+2. Click on an image to open the side menu, and select the **Actions** tab.
+3. In the **Run model** section, select your model and specify a confidence threshold.
+4. Click **Run model**
+
+If the results exceed the confidence threshold, the **Run model** section shows a label and the responding confidence threshold.
+
 ## Deploy an ML model
 
-To use ML models with your machine, you must first deploy the model using an ML model service. The ML model service will run the model and allow the vision service to use it:
+Add the [ML model](/services/ml/) service and deploy your model to your machine to be able to use it to understand your machine's surroundings.
```

Comment:
I think this sentence is more confusing than the original. Also the link doesn't help because it just takes them to ML in general.

```suggestion
To use an ML model on your machine, you need to deploy it to your machine using a compatible ML model service.
```